exporter/opentelemetry-exporter-prometheus-remote-write/README.rst

```diff
@@ -1,27 +1,248 @@
+
+Configuring the Exporter
+------------------------
+
+The exporter can be configured through parameters passed to the
+constructor. Here are all the options:
+
+-  ``endpoint``: url where data will be sent **(Required)**
+-  ``basic_auth``: username and password for authentication
+   **(Optional)**
+-  ``headers``: additional headers for remote write request as
+   determined by the remote write backend's API **(Optional)**
+-  ``timeout``: timeout for requests to the remote write endpoint in
+   seconds **(Optional)**
+-  ``proxies``: dict mapping request proxy protocols to proxy urls
+   **(Optional)**
+-  ``tls_config``: configuration for remote write TLS settings
+   **(Optional)**
+
+Example with all the configuration options:
+
+.. code:: python
+
+   exporter = PrometheusRemoteWriteMetricsExporter(
+       endpoint="http://localhost:9009/api/prom/push",
+       timeout=30,
+       basic_auth={
+           "username": "user",
+           "password": "pass123",
+       },
+       headers={
+           "X-Scope-Org-ID": "5",
+           "Authorization": "Bearer mytoken123",
+       },
+       proxies={
+           "http": "http://10.10.1.10:3000",
+           "https": "http://10.10.1.10:1080",
+       },
+       tls_config={
+           "cert_file": "path/to/file",
+           "key_file": "path/to/file",
+           "ca_file": "path_to_file",
+           "insecure_skip_verify": true, # for developing purposes
+       }
+   )
+
+Securing the Exporter
+---------------------
+
+Authentication
+~~~~~~~~~~~~~~
+
+The exporter provides two forms of authentication which are shown below.
+Users can add their own custom authentication by setting the appropriate
+values in the ``headers`` dictionary
+
+1. Basic Authentication Basic authentication sets a HTTP Authorization
+   header containing a base64 encoded username/password pair. See `RFC
+   7617`_ for more information. This
+
+.. code:: python
+
+   exporter = PrometheusRemoteWriteMetricsExporter(
+       basic_auth={"username": "base64user",  "password": "base64pass"}
+   )
+
+2. Bearer Token Authentication This custom configuration can be achieved
+   by passing in a custom ``header`` to the constructor. See `RFC 6750`_
+   for more information.
+
+.. code:: python
+
+   header = {
+       "Authorization": "Bearer mytoken123"
+   }
+
+TLS
+~~~
+
+Users can add TLS to the exporter's HTTP Client by providing certificate
+and key files in the ``tls_config`` parameter.
+
+Supported Aggregators
+---------------------
+
+-  Sum
+-  MinMaxSumCount
+-  Histogram
+-  LastValue
+-  ValueObserver
+
+Error Handling
+--------------
+
+In general, errors are raised by the calling function. The exception is
+for failed requests where any error status code is logged as a warning
+instead.
+
+This is because the exporter does not implement any retry logic as it
+sends cumulative metrics data. This means that data will be preserved
+even if some exports fail.
```

Comment:
`This means that data will be preserved
even if some exports fail.`

I think this is confusing. It might be interpreted as "my data that was failed to send will be somehow preserved and sent again", which contradicts the previous sentence `does not implement any retry logic`.

Instead of `This means that data will be preserved
even if some exports fail.`, I think it would be good to have `that means data that has failed to export will be dropped`.

Reply:
alright, rephrased the paragraph